app/src/test/java/org/wikipedia/travel/database/DestinationDbHelperTest.java

```diff
@@ -0,0 +1,60 @@
+package org.wikipedia.travel.database;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File does not end with a newline.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)